### ISSUE TITLE
Add "Report an Issue" option in NVDA's Help menu.

### ISSUE THREAD

**Issue by gauravahir28** [nvaccess/nvda#12270]
### Is your feature request related to a problem? Please describe.
Many of NVDA users do not know how to report the issue if they find.
As most of the other applications contains "report an issue" option mostly inside "Help" menu, NVDA also should have the same under "Help" menu so that, more users can report the bugs if any.
This will ensure that more Bugs get identified and addressed as well as it is an open source project.
The more bugs addressed, the more users to start using NVDA.
### Describe the solution you'd like
Add "Report an Issue" option inside NVDA's "Help" menu.

**Comment by DrSooom** [comment]
Duplicate of issue #4033, #7745 and #8685.

**Comment by seanbudd** [comment]
Thanks @gauravahir28 - I agree that this would be a great feature down the line if the existing bug backlog is addressed. There's already discussion on this on #4033, feel free to take a look and leave any additional thoughts that might have been missed in that ticket.
